eventmesh-runtime/conf/eventmesh.properties

```diff
@@ -39,6 +39,9 @@ eventMesh.server.tcp.msgReqnumPerSecond=15000
 eventMesh.server.http.msgReqnumPerSecond=15000
 eventMesh.server.session.upstreamBufferSize=20
 
+eventmesh.server.eventSize=1000
+eventmesh.server.eventBatchSize=10
```

Comment:
Maybe it's better to change to 
```
eventmesh.server.maxEventPerLength=1000
eventmesh.server.maxEventBatchSize=10
```
And add some comments on this config can help user easy to understand.